### ISSUE TITLE
Text resizes with window width when using Windows PowerToys FancyZones 

### ISSUE THREAD

**Issue by lejban** [mintty/mintty#1270]
mintty 3.7.1
Windows 11
Powertools 0.81.1

When resizing using snapping to defined Fancy zones, test scales with the width of the window.

The text size can easily be reset with ctrl + 0 afterwards.

This does not happen when the window is resized with the mouse, nor if I resize use WIN + left/right arrow.

Steps to reproduce:

* Install Microsoft PowerToys - https://learn.microsoft.com/en-us/windows/powertoys/fancyzones
* Enable danzy zones
* Move/resize the windows by holding shift
* Observe text size change in mintty

**Comment by mintty** [comment]
Explanation is easy: with Shift, resizing also zooms the font as described in the manual.
Now if Shift is also part of a hotkey or other external function like in this case, the question is whether this can be distinguished.

**Comment by mintty** [comment]
> Install Microsoft PowerToys - https://learn.microsoft.com/en-us/windows/powertoys/fancyzones

I don't see a download link on that page. How is it installed? And please describe the next two steps in detail for easy reproduction.

**Comment by mintty** [comment]
Yes, that's nice, thank you. And I'm not unwilling to handle this as a special case, if it can be distinguished. First step, however, would be that I'm getting enabled to reproduce the issue, without digging into download and installation issues myself.

**Comment by mintty** [comment]
There is no indication in the Windows messages (checked meanwhile) that would indicate the source of the resize request.
So it cannot be handled specifically.
